docs/development.md

```diff
@@ -261,7 +262,7 @@ To test your changes to thundernetes to a Kubernetes cluster, you can use the fo
 
 - The Makefile on the root of the project contains a variable `NS` that points to the container registry that you use during development. So you'd need to either set the variable in your environment (`export NS=<your-container-registry>`) or set it before calling `make` (like `NS=<your-container-registry> make build push`).
 - Login to your container registry (`docker login`)
-- Run `make build push` to build the container images and push them to your container registry
+- Run `make clean build push` to build the container images and push them to your container registry
```

Comment:
some observations here:
- two subsequent runs of `make build push` incur a 40 second download of modules (I think). Weird that the cached image is not used
- the Dockerfile on the gameserver-api container leads to this image being searched for "push". However, I think that it currently is not built, so it fails

Reply:
also, `NS=docker.io/dgkanatsios/ make clean build push create-install-files-dev` doesn't seem to produce accurate install dev files:
![image](https://user-images.githubusercontent.com/8256138/147187837-fab5de37-e26b-4425-bb19-f97097c603d5.png)

Reply:
Oh, something must have gotten garbled in our feedback changes. I'll double check this again.

Reply:
The double mod download is fixed from the docker line reordering.
The missing image tag was a variable scope issue, which is corrected.